.terraform/modules-under-test.txt

```diff
@@ -0,0 +1,20 @@
+# Modules to be tested when no specific module is given to test.sh as an argument
```

Comment:
I think this file can be replaced by a command to find directories that contain Terraform configuration. With that we can eliminate a possibility of having discrepancy (Terraform files are added to modules but forget updating this file). What do you think?

Reply:
Good suggestion. TODO